lib/apiary/command/styleguide.rb

```diff
@@ -196,11 +222,11 @@ def check_api_key
       abort 'Error: API key must be provided through environment variable APIARY_API_KEY. \Please go to https://login.apiary.io/tokens to obtain it.'
     end
 
-    def load
-      @add_path = api_description_source_path(@options.add)
-      @add = api_description_source(@add_path)
-      @functions = get_functions(@options.functions)
-      @rules = get_rules(@options.rules)
+    def load(add, functions, rules)
+      @add_path = api_description_source_path(@options.add) if add
```

Comment:
Can't this be under one `if`?

Reply:
no - need false, true, true for push

Reply:
I mean there are 2 `if add`

Reply:
ah that one -doesn't matter (+ more lines)

Reply:
![giphy](https://user-images.githubusercontent.com/1788727/38504289-962ce20c-3c14-11e8-823d-079ad1a7a0a6.gif)

It's a code smell